ocm/ocm-overview.adoc

```diff
@@ -0,0 +1,44 @@
+[id="ocm-overview"]
+= Red Hat OpenShift Cluster Manager (OCM)
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: ocm-overview
+toc::[]
+
+The Red Hat OpenShift Cluster Manager (OCM) allows users to create, manage, and delete OpenShift clusters using a web UI.
```

Comment:
Can we add a little bit more introductory context here? Something like (feel free to wordsmith):

_OpenShift Cluster Manager is a managed service where you can install, modify, operate and upgrade your Red Hat OpenShift clusters. It allows you to work with all of your organization’s clusters from single dashboard.  OpenShift Cluster Manager provides the links and steps to install OpenShift Container Platform, ROSA and OSD clusters and is responsible for managing both OpenShift Container Platform clusters after self-installation and your ROSA and OSD clusters.

From OpenShift Cluster Manager, you can:
- Create new clusters
- View cluster details and metrics
- Manage your clusters (scale, node labels, networking, authentication, etc)
- Manage access control
- Monitor clusters
- Schedule upgrades_

(Above modified from here: https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-what-is-ocm)